subsys/bluetooth/mesh/pb_gatt_srv.c

```diff
@@ -277,11 +277,12 @@ int bt_mesh_pb_gatt_srv_adv_start(void)
 	};
 	struct bt_data prov_sd[1];
 	size_t prov_sd_len;
-	int err;
+	int64_t timestamp = fast_adv_timestamp;
+	int64_t elapsed_time = k_uptime_delta(&timestamp);
 
 	prov_sd_len = gatt_prov_adv_create(prov_sd);
 
-	if (!prov_fast_adv) {
+	if (elapsed_time > 60ll * MSEC_PER_SEC) {
```

Comment:
Since you have to repeat the `60 * MSEC_PER_SEC` in a couple of places, I think it'd make the code more readable if you just added a `#define FAST_ADV_TIME (60 * MSEC_PER_SEC)`.